tools/exporter/exporter.py

```diff
@@ -51,6 +51,9 @@
 # When exporting actions to CSV, if property is None, print this value to CSV
 EMPTY_RESPONSE = ''
 
+# If user specifies this for the export filename, the audit_data.name property is used.
```

Comment:
Not all Audits will actually contain an Audit Title item. For examples, when Audit Title rules are set, the Audit Title item is not going to be included by default. When this item ID is specified in the custom export filename configuration, the `audit_data.name` property will be used to populate the data as it covers all cases.